### PR TITLE
Update Enable-VMMigration.md

### PR DESCRIPTION
**Pull request by msbemba**
Added example with the computer name attribute

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/windows-powershell-docs/issues/1793

**Comment by e0i** [comment]
@dariomws @get-itips @tseward

Peer review needed for this PR. Thanks.

File: docset/windows/hyper-v/Enable-VMMigration.md [hunk 1]
```diff
@@ -48,6 +48,13 @@ PS C:\> Enable-VMMigration
 
 Enables live migration on the local Hyper-V host.
 
+### Example 2
+```
+PS C:\> Enable-VMMigration -Comptername TestVM -Passthru
```
Comment (get-itips) [inline docset/windows/hyper-v/Enable-VMMigration.md:53]:
```suggestion
PS C:\> Enable-VMMigration -ComputerName TestVM -Passthru 
```

File: docset/windows/hyper-v/Enable-VMMigration.md [hunk 2]
```diff
@@ -48,6 +48,13 @@ PS C:\> Enable-VMMigration
 
 Enables live migration on the local Hyper-V host.
 
+### Example 2
+```
+PS C:\> Enable-VMMigration -Comptername TestVM -Passthru 
+```
+
+When Trying to migrate a VM, the host needs to be enabled for VMmigration and the exmaple Enables the TestVM(host) for Migration.
```
Comment (get-itips) [inline docset/windows/hyper-v/Enable-VMMigration.md:56]:
```suggestion
Enables live migration on the specified Hyper-V host. 
```

**Comment by e0i** [comment]
@tseward Good to go with your approval. Thank you.

**Pull request merged**